src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -201,49 +201,30 @@ Unix support is disabled by default. See https://aka.ms/systemdrawingnonwindows
   <!-- Windows-specific -->
   <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
```

Comment:
I got confused over what to do with these items' conditions. This is always true and the same applies with the one on line 33? But we are also building editions that don't contain any code.

Reply:
When building for `netstandard2.0`, `TargetsWindows` would be false. The assembly in this case will just every public member, each throwing PlatformNotSupportedException.

Reply:
I see, and the `$(TargetsAnyOS) != true` above used to be true only on `net(6|7).0-(windows|unix)`, and now that Unix is gone, only Windows remains and the two groups can be merged. Done.

Reply:
Exactly.